L1TriggerScouting/OnlineProcessing/plugins/BMTFStubMultiBxSelector.cc

```diff
@@ -0,0 +1,170 @@
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "FWCore/Utilities/interface/StreamID.h"
+
+// L1 scouting
+#include "DataFormats/L1Scouting/interface/L1ScoutingBMTFStub.h"
+#include "DataFormats/L1Scouting/interface/OrbitCollection.h"
+#include "L1TriggerScouting/Utilities/interface/conversion.h"
+
+#include <iostream>
```

Comment:
` #include <iostream>` is unused

Reply:
Removed as suggested. Removed also
```
#include "L1TriggerScouting/Utilities/interface/conversion.h"
```
which not used